src/Hyperion.Akka.Integration.Tests/Hyperion.Akka.Integration.Tests.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Probably need to have this reference `common.props`

Reply:
Good point.